test/fixtures/test-runner/output/test-runner-watch-spec.snapshot

```diff
@@ -0,0 +1,30 @@
+✖ failing test (*ms)
+✔ test has ran (*ms)
+ℹ tests 2
+ℹ suites 0
+ℹ pass 1
+ℹ fail 1
+ℹ cancelled 0
+ℹ skipped 0
+ℹ todo 0
+ℹ duration_ms *
+
+✖ failing tests:
+
+*
+✖ failing test (*ms)
+  Error: failed
+      *
+      *
+      *
+      *
+      *
+      *
+ℹ tests 0
+ℹ suites 0
+ℹ pass 0
+ℹ fail 0
+ℹ cancelled 0
+ℹ skipped 0
+ℹ todo 0
```

Comment:
Why are we generating a summary with nothing executed?

Reply:
Yes, but unfortunately, this is not related to the changes in this PR.  
This probably never came up because it only happens when a watcher is stopped via abort